userspace/hardware_setup.sh

```diff
@@ -2,6 +2,11 @@
 
 # Install driver deb files (we're fine with overwriting stuff too)
 cd /tmp/agnos/debs
+
+# Fixing issue with missing /lib64
```

Comment:
what's relying on this?

Reply:
24.04 has [UsrMerge](https://wiki.debian.org/UsrMerge) - 20.04 didn't.

I need to create `usrmerged /lib64` before `agnos-base.deb` installs a file in `/lib64`.

`/lib64` in 20.04 was a regular folder. In 24.04 it needs to be `usrmerged`, aka `/lib64` is a symlink to `/usr/lib64`.

If I don't create the symlink, it fails with an error that is not usrmerged. I can remove the comment and/or try to update `agnos-base.deb`.